ext/random/tests/03_randomizer/methods/getBytes.phpt

```diff
@@ -27,7 +27,7 @@ foreach ($engines as $engine) {
     $randomizer = new Randomizer($engine);
 
     // Using 10_000 is very slow.
-    for ($i = 1; $i < 1_000; $i++) {
+    for ($i = 0; $i < 1_000; ++$i) {
```

Comment:
```suggestion
    for ($i = 0; $i < 1_000; $i++) {
```

As discussed in #12279, please don't make unrelated adjustments to the tests.

Reply:
@TimWolla i thought it was just someone accidentally using the wrong operator, so i fixed it. this was added before i knew there was an official policy of deliberately using the wrong operator. reverted.